transport/grpc/client.go

```diff
@@ -70,6 +70,7 @@ func SetClientBefore(before ...RequestFunc) ClientOption {
 // client.
 func (c Client) Endpoint() endpoint.Endpoint {
 	return func(ctx context.Context, request interface{}) (interface{}, error) {
+		mdWithApiKey, hasApiKey := metadata.FromOutgoingContext(ctx)
```

Comment:
move this to line 87

Reply:
actually, I cannot move it to line 86 as the `ctx` is being overridden by line 74.